.gitmodules

```diff
@@ -25,3 +25,6 @@
 [submodule "ThirdParty/vtkGUISupportQt"]
 	path = ThirdParty/vtkGUISupportQt
 	url = https://github.com/ufz/vtkGUISupportQt.git
+[submodule "ThirdParty/metis"]
+	path = ThirdParty/metis
+	url = https://github.com/envinf/metis.git
```

Comment:
It will be committed a to ufz-github.
There is only one three lines added in mpmetis.c as
```
#ifdef BUILD_WITH_METIS
#include "../../../Applications/Utils/ModelPreparation/PartitionMesh/metis_main.h"
int metis_main(int argc, char *argv[])
#else
int main(int argc, char *argv[])
#endif
``` 
in order to call this main function. Alternatively, I can keep the source code untouched, and let the code execute 'metis' externally.